features/networking/multus.feature

```diff
@@ -991,3 +991,36 @@ Feature: Multus-CNI related scenarios
     When I execute on the pod:
       | /usr/sbin/ip | a |
     Then the output should contain "192.168.1"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-25909
+  @admin
+  Scenario: Assign static IP address using pod annotation			 
```

Comment:
please help delete the blank at the end.

Reply:
Fixed the blank issue